<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PathCheckFinderWgt</class>
 <widget class="QWidget" name="PathCheckFinderWgt">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>23</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="lbl_path">
     <property name="text">
      <string>Path</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="le_path"/>
   </item>
   <item>
    <widget class="QLabel" name="lbl_check_icon">
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="../../resources/icons/small/small_icons.qrc">:/icons/small/check.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="tb_find">
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="../../resources/icons/small/small_icons.qrc">
       <normaloff>:/icons/small/magnifying-glass.png</normaloff>:/icons/small/magnifying-glass.png</iconset>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../resources/icons/small/small_icons.qrc"/>
 </resources>
 <connections/>
</ui>
